visualization/FairnessDashboard/src/Localization/en.json

```diff
@@ -93,17 +109,22 @@
         "distributionOfPredictions": "Distribution of predictions",
         "distributionOfErrors": "Distribution of errors",
         "tooltipPrediction": "Prediction: {0}",
-        "tooltipError": "Error: {0}"
+        "_tooltipPrediction.comment": "Displays tooltip with the formatted numerical value of the prediction",
+        "tooltipError": "Error: {0}",
+        "_tooltipError.comment": "Displays tooltip with the formatted numerical value of the error"
     },
     "Feature": {
         "header": "Along which features would you like to evaluate your model's fairness?",
         "body": "Fairness is evaluated in terms of disparities in your model's behavior. We will split your data according to values of each selected feature, and evaluate how your model's performance metric and predictions differ across these splits.",
         "learnMore": "Learn more",
         "summaryCategoricalCount": "This feature has {0} unique values",
+        "_summaryCategoricalCount.comment": "Number of unique values of the feature",
         "summaryNumericCount": "This numeric feature ranges from {0} to {1}, and is grouped into {2} bins.",
+        "_summaryNumericalCount.comment": "The numerical range (low and high values) of the feature, and number of bin groups within that range",
         "showCategories": "Show all",
         "hideCategories": "Collapse",
         "categoriesOverflow": "   and {0} additional categories",
+        "_categoriesOverflow.comment": "??? NOT IN SRC - number of remaining additional categories",
```

Comment:
categoriesOverflow not found in /src
@rihorn2 ok to remove?

Reply:
Interesting, it had been used to prevent listing too many categories, but if its not in the source, then I guess it was removed from the design?